CIP-0030/README.md

```diff
@@ -368,6 +374,10 @@ Extensions can be seen as a smart versioning scheme. Except that, instead of bei
 2. Not everyone agrees and has desired to support every existing standard;
 3. There's a need from an API consumer standpoint to clearly identify what features are supported by providers.
 
+#### Namespacing Extension
```

Comment:
For me, it is unclear how a DApp will use these namespaced methods.
Let's take signTx usage as an example:
1. DApp enables cip-95 and cip-1234.
2. Builds a transaction with Conway fields (e.g. registers a DRep certificate) + cip-1234-specific-stuff
Which signTx will the DApp use?
1. cip95.signTx(tx, partialSign=true) + cip1234.signTx(tx, partialSign=true)
2. Split it into 2 transactions and sign them separately?
3. Will all namespaced signTx [extend the cip30 base sign](https://github.com/cardano-foundation/CIPs/blob/2bf97016b1513b497d404e8a3611238eea636944/CIP-0095/README.md#apicip95signtxtx-cbortransaction-partialsign-bool--false-promisecbortransaction_witness_set)?

Reply:
This is a good question!

In this case I believe that

Reply:
This is a good question! and I think the answer would depend on how CIP-1234 defines it's `signTx`.

For me, I think it is best that extensions completely replace other extensions capabilities rather than dApps needing two extensions to do the same thing; such as signing transactions. So my preferred approach would be for CIP-1234 to be supporting all of the CIP-95 capabilities and it's own at once. This would prevent the need for dApps to support two extensions for the same purpose.

With that said, CIP-30 as it stands today would does allow for such awkward overlaps of capabilities between extensions and it is up to wallets to reconcile their capabilities.